src/components/views/elements/AppTile.tsx

```diff
@@ -130,10 +134,25 @@ export default class AppTile extends React.Component<IProps, IState> {
         }
 
         this.state = this.getNewState(props);
+        this.watchUserReady();
 
         this.allowedWidgetsWatchRef = SettingsStore.watchSetting("allowedWidgets", null, this.onAllowedWidgetsChange);
     }
 
+    private watchUserReady = () => {
+        if (OwnProfileStore.instance.isProfileInfoFetched) {
+            return;
+        }
+        OwnProfileStore.instance.on(UPDATE_EVENT, this.onUserReady);
```

Comment:
can this not be a `once` instead of `on` to skip the removeListener in the handler? It seems like the first/any state update in the OwnProfileStore cause `isProfileInfoFetched` to become true.

Reply:
good idea, done.